spec/api/v1/external_users/claim_spec.rb

```diff
@@ -24,6 +24,9 @@
 
     /api/external_users/claims/transfer
     /api/external_users/claims/transfer/validate
+
+    /api/external_users/claims/hardship
```

Comment:
Should this be changed to`/api/external_users/claims/litigators/hardship`?

I know the plan was to move them over to the new namespace eventually, @jsugarman should probably weigh in on this one as it will affect you more than me!

Reply:
yeah :( the whole litigator side should really be name-spaced to:
```
/api/external_users/claims/litigators/[type]
/api/external_users/claims/litigators/[type]/validate
```

but we can't get rid of the old ones. we should probably only be introducing new endpoints in the new namespaces. same applies to AGFS branch for hardship - have not checked this on that branch.

Reply:
actually looking at current API the whole LGFS side is not namespaced. so would be a bit of tangent to do now and sunset the old endpoints. Tech debt 😢 coming back to bite us

<img width="1048" alt="Screen Shot 2020-05-06 at 10 19 58" src="https://user-images.githubusercontent.com/7016425/81161417-74208f00-8f83-11ea-9d39-b390948ce25f.png">

question is whether we should namespace just the lgfs hardship now?!

Reply:
Hence why I asked! 😃 Do you want a single, 'correctly' namespaced litigator endpoint as a forerunner or add more tech debt?  🤔

Reply:
could add the correct one for hardship and the others and it might prompt the vendors to correct the others while they are at it :dreamon:

Reply:
Happy to create a namespaced endspoint for litigator hardship claims or not 😃 I originally added it to the non-namespaced endpoint as that is where all the other litigator endspoints were.

Reply:
lets talk if over, but i am minded to add the namespaced version only for hardship and if @kmahern has time create a separate branch of off this branch that spikes swapping all the others over and deprecating the old versions.

Reply:
👍 I have created added the /api/external_users/claims/litigators/hardship and /api/external_users/claims/litigators/hardship/validate endpoints and a ticket for moving the other litigator claim endpoints (https://dsdmoj.atlassian.net/browse/CBO-1225)